java/src/jmri/jmrix/loconet/streamport/LnStreamPortController.java

```diff
@@ -59,7 +59,7 @@ public void configure() {
         this.getSystemConnectionMemo().setLnTrafficController(packets);
         // do the common manager config
         this.getSystemConnectionMemo().configureCommandStation(commandStationType,
-                mTurnoutNoRetry, mTurnoutExtraSpace, false); // never transponding
+                mTurnoutNoRetry, mTurnoutExtraSpace, false, false); // never transponding
```

Comment:
This should be configurable as well.

Reply:
Apart from Z21 extending it I cannot find where its used.

Reply:
@Pugwash1 @pabender @bobjacobsen 
> Apart from Z21 extending it I cannot find where its used.

There is a `jmri.jmrix.loconet.streamport.LnStreamConnectionConfig` that's using it, but that `LnStreamConnectionConfig` is not used. I don't know why the `LnStreamConnectionConfig` exists. Maybe it's used as a template for new connection configs?

But for me, it seems that `LnStreamPortController` should follow the same pattern as other LocoNet controllers.

But if the rest of you do not agree, I'm happy to approve this PR.

Reply:
> @Pugwash1 @pabender @bobjacobsen
> 
> > Apart from Z21 extending it I cannot find where its used.
> 
> There is a `jmri.jmrix.loconet.streamport.LnStreamConnectionConfig` that's using it, but that `LnStreamConnectionConfig` is not used. I don't know why the `LnStreamConnectionConfig` exists. Maybe it's used as a template for new connection configs?

To answer the question on where else the stream port controllers can be used, in addition to the Z21…

When you have an XBee connection configured, each remote XBee has a serial port that can be connected to another device.  You can use the single XBee connection to multiplex remote serial devices through a single XBee attached to the computer.

I have used this to connect RFID, XPressNet, Loconet, and an SPROG to a computer for various projects.

I have some info on the hardware side of this here:

https://www.nscaleintermodal.com/XBee-MRR/TransparentMode.html#serialreplace

And my RFID test/demo layout uses the RFID and SPROG connections. ( in addition to raw IO pins in the XBees):
https://github.com/pabender/Sensys2014